transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -315,7 +315,7 @@ static jstring netty_epoll_native_kernelVersion(JNIEnv* env, jclass clazz) {
 }
 
 static jboolean netty_epoll_native_isSupportingSendmmsg(JNIEnv* env, jclass clazz) {
-    if (sendmmsg) {
+    if (&sendmmsg) {
```

Comment:
@carl-mastrangelo I am a bit confused... so if you change it to use `&` will this still work as what we tried to do ?